OpenRA.Mods.RA/Widgets/Logic/Ingame/LeaveMapLogic.cs

```diff
@@ -96,6 +96,20 @@ public LeaveMapLogic(Widget widget, World world, OrderManager orderManager)
 				});
 			};
 
+			// if player is alone on map (bots excluded), player got the possibility to restart
+			var restartButton = dialog.Get<ButtonWidget>("RESTART_BUTTON");
+			restartButton.IsVisible = () => !isMultiplayer;
+			restartButton.OnClick = () =>
+			{
+				restartButton.IsDisabled = () => true;
+
+				// call restart
+				Game.RestartGame();
+
+				// close menue
```

Comment:
Guess because the source code has many redundant source code comments with spelling mistakes in it.